tools/SourceKit/tools/sourcekitd/bin/InProc/sourcekitdInProc.exports

```diff
@@ -0,0 +1,82 @@
+sourcekitd_cancel_request
+sourcekitd_initialize
+sourcekitd_request_array_create
+sourcekitd_request_array_set_int64
+sourcekitd_request_array_set_string
+sourcekitd_request_array_set_stringbuf
+sourcekitd_request_array_set_uid
+sourcekitd_request_array_set_value
+sourcekitd_request_create_from_yaml
+sourcekitd_request_description_copy
+sourcekitd_request_description_dump
+sourcekitd_request_dictionary_create
+sourcekitd_request_dictionary_set_int64
+sourcekitd_request_dictionary_set_string
+sourcekitd_request_dictionary_set_stringbuf
+sourcekitd_request_dictionary_set_uid
+sourcekitd_request_dictionary_set_value
+sourcekitd_request_int64_create
+sourcekitd_request_retain
+sourcekitd_request_release
+sourcekitd_request_string_create
+sourcekitd_request_uid_create
+sourcekitd_response_description_copy
+sourcekitd_response_description_dump
+sourcekitd_response_description_dump_filedesc
+sourcekitd_response_dispose
+sourcekitd_response_error_get_description
+sourcekitd_response_error_get_kind
+sourcekitd_response_get_value
+sourcekitd_response_is_error
+sourcekitd_send_request
+sourcekitd_send_request_sync
+sourcekitd_set_interrupted_connection_handler
+sourcekitd_set_notification_handler
+sourcekitd_set_uid_handler
+sourcekitd_set_uid_handlers
+sourcekitd_shutdown
+sourcekitd_uid_get_from_buf
+sourcekitd_uid_get_from_cstr
+sourcekitd_uid_get_length
+sourcekitd_uid_get_string_ptr
+sourcekitd_variant_array_apply
+sourcekitd_variant_array_apply_f
+sourcekitd_variant_array_get_bool
+sourcekitd_variant_array_get_count
+sourcekitd_variant_array_get_int64
+sourcekitd_variant_array_get_string
+sourcekitd_variant_array_get_uid
+sourcekitd_variant_array_get_value
+sourcekitd_variant_bool_get_value
+sourcekitd_variant_description_copy
+sourcekitd_variant_description_dump
+sourcekitd_variant_description_dump_filedesc
+sourcekitd_variant_dictionary_apply
+sourcekitd_variant_dictionary_apply_f
+sourcekitd_variant_dictionary_get_bool
+sourcekitd_variant_dictionary_get_int64
+sourcekitd_variant_dictionary_get_string
+sourcekitd_variant_dictionary_get_value
+sourcekitd_variant_dictionary_get_uid
+sourcekitd_variant_get_type
+sourcekitd_variant_json_description_copy
+sourcekitd_variant_string_get_length
+sourcekitd_variant_string_get_ptr
+sourcekitd_variant_data_get_size
+sourcekitd_variant_data_get_ptr
+sourcekitd_variant_int64_get_value
+sourcekitd_variant_uid_get_value
+_ZN10sourcekitd13enableLoggingEN4llvm9StringRefE
```

Comment:
I copied the `sourcekitd_` symbols here, because I don't see a way to do something like `#include`.  What do you think of changing this to use a wildcard like `sourcekitd_*`?  I can't think of a good reason we would have an `extern "C"` symbol with `sourcekitd_` prefix unless we wanted to export it.

Reply:
SGTM 👍

Reply:
This didn't end up working. Windows doesn't support the wildcard.